core/src/Log.purs

```diff
@@ -98,10 +98,14 @@ instance Loggable String where
 instance Loggable PackageName where
   toDoc = PackageName.print >>> toDoc
 
-instance Loggable a => Loggable (Array a) where
+instance Loggable (Array Docc) where
+  toDoc = Log.lines
+else instance Loggable a => Loggable (Array a) where
   toDoc = Log.lines <<< map toDoc
 
-instance Loggable a => Loggable (NonEmptyArray a) where
+instance Loggable (NonEmptyArray Docc) where
+  toDoc = Log.lines <<< NEA.toArray
+else instance Loggable a => Loggable (NonEmptyArray a) where
```

Comment:
Why these instance chains? They seem unnecessary

Reply:
It just removes an unnecessary `map` when we know what the `a` is. I've removed them.